### PR TITLE
GSK-206 fix pytorch model

### PR DESCRIPTION
**Pull request by princyiakov**
## Description

PR to fix the bug of pytorch demo model .  Previous demo model was accepting series instead of dataframe which lead to error in inspect .  

## Type of Change

- [ ✅] 🔧 Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: giskard-demo-notebook/notebook/Email Classification Model.ipynb [hunk 1]
```diff
@@ -909,51 +909,49 @@
...
      "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
     "def predict(test_dataset):\n",
+    "    original_ds_len = len(test_dataset)\n",
+    "    assert type(test_dataset) == pd.core.frame.DataFrame\n",
+    "    test_dataset= test_dataset.squeeze() if len(test_dataset) >1 else test_dataset\n",
+    "    X_test = list(test_dataset)\n",
```
Comment (andreybavt) [inline giskard-demo-notebook/notebook/Email Classification Model.ipynb:922]: out of curiosity, it doesn't work with pandas dataframe?

**Review by andreybavt — CHANGES_REQUESTED** [review]

File: giskard-demo-notebook/notebook/Email Classification Model.ipynb [hunk 2]
```diff
@@ -909,51 +909,49 @@
...
-    "id": "cTuCGaP2ZZnI",
     "pycharm": {
      "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
     "def predict(test_dataset):\n",
+    "    original_ds_len = len(test_dataset)\n",
+    "    assert type(test_dataset) == pd.core.frame.DataFrame\n",
```
[the remark this answers is not in the document]
Comment (princyiakov) [reply giskard-demo-notebook/notebook/Email Classification Model.ipynb:920]: Infact I had kept it intentionally for user to be sure that their model works even before uploading on Giskard and doing a re-work  . WDYT ?

File: giskard-demo-notebook/notebook/Email Classification Model.ipynb (hunk 1 above)
Comment (princyiakov) [reply giskard-demo-notebook/notebook/Email Classification Model.ipynb:922]: It was giving. wrong results for the pandas dataframe :(

File: giskard-demo-notebook/notebook/Email Classification Model.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (andreybavt) [reply giskard-demo-notebook/notebook/Email Classification Model.ipynb:920]: But if we add this check to `upload_model` like the other validations a user will still get an error before the model is uploaded to Giskard. Having this check outside of `upload_model` would mean that if a user doesn't check it himself it won't be checked.

**Review by andreybavt — CHANGES_REQUESTED** [review]

File: giskard-demo-notebook/notebook/Email Classification Model.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (princyiakov) [reply giskard-demo-notebook/notebook/Email Classification Model.ipynb:920]: That is true indeed . I ll update the notebook

**Review by andreybavt — APPROVED** [review]

**Pull request merged**